app/controllers/application_controller.rb

```diff
@@ -4,6 +4,7 @@ class ApplicationController < ActionController::Base
   before_action :set_current_user
   rescue_from Faraday::ClientError, with: :api_error
   rescue_from Octokit::Unauthorized, with: :github_unauthorized_error
+  rescue_from Octokit::ServiceUnavailable, with: :api_error
```

Comment:
Maybe we should recuse from `Octokit::ServerError` instead, so we cleanly handle all 5xx that GitHub might throw at us?

What do you think about handling other client errors from Octokit, or are we happy to let the app do what it does currently for those until we see an issue arrise?

Reply:
Yeah, makes sense to handle all 5xx –– let's do that.

As far as the other errors –– I wish we had more data from last year! Let me at least add `Octokit::AccountSuspended`, which was mentioned in #431